src/PIL/PngImagePlugin.py

```diff
@@ -981,7 +981,14 @@ def load_end(self):
             except EOFError:
                 if cid == b"fdAT":
                     length -= 4
-                ImageFile._safe_read(self.fp, length)
+                try:
+                    ImageFile._safe_read(self.fp, length)
+                except OSError as e:
+                    if ImageFile.LOAD_TRUNCATED_IMAGES:
+                        logger.debug("%r %s %s (Truncated File Read)", cid, pos, length)
```

Comment:
```suggestion
```
I don't think you need the logging, because none of code that deals with truncated code in ImageFile or PngImagePlugin has logging - but feel free to disagree with me.